### PR TITLE
Add Node.js example

### PR DESCRIPTION
**Pull request by marczwalua**
Created Node.js example with http-server

### CODE REVIEW

File: application/nodejs/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "nodejs-secrethub-demo",
+  "version": "1.0.0",
+  "description": "nodejs-secrethub-demo",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js"
+  },
+  "author": "SecretHub",
+  "license": "ISC",
```
Comment (florisvdg) [inline application/nodejs/package.json:10]: I'd say let's do `MIT`.

Comment (marczwalua) [reply application/nodejs/package.json:10]: Sure, I'll change it.

Comment (marczwalua) [reply application/nodejs/package.json:10]: And the comment about the package-json.lock, if it does not exist, there will be a warning. Another option is to touch the file with Docker, but I thought this was a nice solution.

Comment (florisvdg) [reply application/nodejs/package.json:10]: NPM should automatically populate the `package-lock.json`, didn't it do that? Or did you remove it?

Comment (marczwalua) [reply application/nodejs/package.json:10]: Yes you're right. npm does create automaticly. I'll remove package-lock.json

Comment (florisvdg) [reply application/nodejs/package.json:10]: No I meant: include it. Because it increases portability.

Comment (marczwalua) [reply application/nodejs/package.json:10]: Ah, so include it and leave it empty like it was?

Comment (florisvdg) [reply application/nodejs/package.json:10]: No, not empty, but populated with the contents it autogenerates when you run `npm install`.

Comment (marczwalua) [reply application/nodejs/package.json]: Sure, I'll add it

**Comment by florisvdg** [comment]
The link in the index README is still missing though.

**Pull request merged**